client/src/components/ChatMessage.tsx

```diff
@@ -826,19 +962,7 @@ export function translateChatMessage(
                 translateChecked(`chatMessage.gameOver.conclusion.${message.synopsis.conclusion}`)
                 ?? translate(`chatMessage.gameOver.conclusion.unknown`, translateConclusion(message.synopsis.conclusion))
             
-            return conclusionString + '\n'
-                + message.synopsis.playerSynopses.map((synopsis, index) => 
-                    translate(`chatMessage.gameOver.player.won.${synopsis.won}`, encodeString(playerNames![index]))
-                        + ` (${synopsis.outlineAssignment + 1}: ${translateRoleOutline(roleList[synopsis.outlineAssignment], playerNames)}`
-                        + `: ${synopsis.crumbs.map(crumb => 
-                            translate("chatMessage.gameOver.player.crumb",
-                                crumb.insiderGroups.map(group => translate("chatGroup."+group+".icon")).join("|") || translate("chatGroup.all.icon"),
-                                translateWinCondition(crumb.winCondition), 
-                                translate(`role.${crumb.role}.name`)
-                            )
-                        ).join(" → ")}`
-                        + `)`
-                ).join('\n');
+            return conclusionString;
```

Comment:
This should include at LEAST all the players, if not all the information provided by the synopsis, for filtering purposes.